assets/js/components/notifications/UserInputSettings.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * UserInputSettings component.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Notification from './notification';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import personSitImage from '../../../images/person-sit.png';
+const { useSelect } = Data;
+
+export default function UserInputSettings() {
+	const ctaLink = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-user-input' ) );
+	const userInputState = useSelect( ( select ) => select( CORE_USER ).getUserInputState() );
+
+	if ( userInputState === 'completed' ) {
+		return null;
+	}
+
+	return (
+		<Notification
+			id="user-input-settings-notification"
```

Comment:
Let's use [`useInstanceId`](https://developer.wordpress.org/block-editor/packages/packages-compose/#useInstanceId) or something similar to make sure this ID is unique. I think it's unlikely we'll have multiple of this component on the same page, but just in case: let's make sure it's ensured to be unique.

Reply:
Going with `useInstanceId` rather than something else as we're already using `withInstanceId` in a few cases which is the higher-order component version (we're moving away from HOCs in favor of hooks, seems we could replace the others as a future tech debt cleanup).